src/v1/core.ts

```diff
@@ -18,8 +18,9 @@ import {
 } from './interfaces.compatible';
 import * as v20220901 from './interfaces.v20220901';
 import * as v20221101 from './interfaces.v20221101';
+import * as v20230901 from './interfaces.v20230901';
 
-type APIVersion = '2022-09-01' | '2022-11-01';
+type APIVersion = '2022-09-01' | '2022-11-01' | '2023-09-01';
```

Comment:
Consider using a switch case or a mapping object for handling different API versions. This can improve the readability and maintainability of the code. [medium]